dcfs/dcfs-share/dcfs-run/demo.py

```diff
@@ -122,6 +122,8 @@ def send_task_status(task_id, status, message, link='', table_name=''):
                 logging.info("Retrieving data from MySQL")
                 send_task_status(task_id, TASKSTATUS_PROCESSING, "Retrieving data from MySQL")
                 locals()['df%d'%i] = pd.read_sql(d['sql'], con=db_engine)
+                db_engine.connect().close()
```

Comment:
Hello @chungen0126,
I don't really get this.

In this line, do you mean to initiate a new connection to the DBMS, and then close the connection immediately?

ref:
https://docs.sqlalchemy.org/en/13/core/connections.html#sqlalchemy.engine.Engine.connect

Reply:
I think only 
```suggestion
                db_engine.dispose()
```
is needed?

ref: https://stackoverflow.com/a/47964066/16113175